e2e/cypress/integration/multi_team_and_dm/close_current_dm_redirects_spec.js

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+describe('Direct messages: redirections', () => {
+    let testUser;
+    let secondDMUser;
+    let firstDMUser;
+    let testTeam;
+    let offTopicChannelUrl;
+
+    before(() => {
+        // # Create a new team
+        cy.apiInitSetup().then(({team, user}) => {
+            testUser = user;
+            testTeam = team;
+
+            offTopicChannelUrl = `/${testTeam.name}/channels/off-topic`;
+
+            cy.apiCreateUser().then(({user: createdUser}) => {
+                firstDMUser = createdUser;
+                cy.apiAddUserToTeam(team.id, firstDMUser.id);
+            });
+
+            // # Create a second test user
+            cy.apiCreateUser().then(({user: createdUser}) => {
+                secondDMUser = createdUser;
+                cy.apiAddUserToTeam(testTeam.id, secondDMUser.id);
+            });
+        });
+    });
+
+    it('MM-T453 Closing a direct message should redirect to last viewed channel', () => {
+        // # Login as test user
+        cy.apiLogin(testUser);
+
+        // # View 'off-topic' channel
+        cy.visit(offTopicChannelUrl);
```

Comment:
Same here. Could be move to `beforeEach`?

Reply:
I agree. We should put this block to beforeEach and remove from individual tests, unless we have other test cases to be added that won't comply to the pattern, but we can refactor later.